awx/main/analytics/core.py

```diff
@@ -220,13 +223,17 @@ def _write_manifest(destdir, manifest):
         logger.exception("Failed to write analytics archive file")
     finally:
         shutil.rmtree(dest, ignore_errors = True)
+    stop_gather = time.perf_counter()
+    logger.info(f"Stop gather - duration: {stop_gather - start_gather}")
```

Comment:
May want to lower this to debug unless we want this in the logs every time. @ryanpetrello ?

Reply:
Yes, these should both be `.debug()`.

Reply:
Thank you and I just made changes given on comments.